virtualBridge.py

```diff
@@ -162,10 +165,21 @@ def configure(self) -> None:
             # Without this, net-undefine within ensure_bridge_is_started fails as libvirtd fails to restart
             # We need to investigate how to remove the sleep to speed up
             time.sleep(5)
-            self._ensure_started(bridge_xml)
+            self._ensure_started(bridge_xml, api_port)
 
             self._restart()
 
             # Not sure why/whether this is needed. But we saw failures w/o it.
             # We need to investigate how to remove the sleep to speed up
             time.sleep(5)
+
+        # Detect vir bridge mac address.
+        bridge_port = common.find_port(self.hostconn, 'virbr0')
+        self._eth_address = bridge_port.address if bridge_port is not None else None
```

Comment:
There's a bug in this PR. virbr eth address detection should retry in case the bridge is still configuring. Otherwise we might end up with:

AttributeError: 'VirBridge' object has no attribute '_eth_address'

I'll fix it.

Reply:
Should be fixed now.